.circleci/config.yml

```diff
@@ -36,7 +36,7 @@ commands:
         type: string
     steps:
       - checkout
-      - run: GOOS=<< parameters.GOOS >> go build -ldflags="-s -w" -o ./pkg/packer_<< parameters.GOOS >>_$(go env GOARCH) .
+      - run: GOOS=<< parameters.GOOS >> go build -ldflags="-s -w -X github.com/hashicorp/packer/version.GitCommit=${CIRCLE_SHA1}" -o ./pkg/packer_<< parameters.GOOS >>_$(go env GOARCH) .
```

Comment:
Turn out there was already a var doing this:

https://github.com/hashicorp/packer/blob/59fa6c036c454022b0a6d6a785f8f0b4de5af14b/version/version.go#L7-L10